packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
+  justify-content: space-between;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-xsmall);
+  width: 100%;
```

Comment:
question: do we really need it?

Reply:
Yes, the meter has to adapt to the context in which it is used, we cannot define a fixed value for it, it generally occupies 100% of an element